library/Notifications/Model/Contact.php

```diff
@@ -7,10 +7,24 @@
 use Icinga\Module\Notifications\Model\Behavior\HasAddress;
 use ipl\Orm\Behaviors;
 use ipl\Orm\Model;
+use ipl\Orm\Query;
 use ipl\Orm\Relations;
 
 /**
  * @property int $id
+ * @property string $full_name
+ * @property ?string $username
+ * @property string $color
+ * @property int $default_channel_id
+ *
+ * @property Channel | Query $channel
+ * @property ContactAddress | Query $contact_address
+ * @property Incident | Query $incident
+ * @property IncidentContact | Query $incident_contact
+ * @property IncidentHistory | Query $incident_history
+ * @property RuleEscalationRecipient | Query $rule_escalation_recipient
+ * @property RotationMember | Query $rotation_member
+ * @property ContactAddress | Query $contactgroup
```

Comment:
This is optional, Most of the existing and upcoming annotations are defined as following:
`@property Query|Channel $channel`. (without a space between `|`)
This way the code looks consistent.